lms/djangoapps/mobile_api/courses/views.py

```diff
@@ -0,0 +1,126 @@
+"""
+Views for courses info API
+"""
+
+
+from rest_framework import generics, permissions, status
+from rest_framework.authentication import OAuth2Authentication, SessionAuthentication
+from rest_framework.response import Response
+from courseware.access import is_mobile_available_for_user
+from student.models import CourseEnrollment
+from social.apps.django_app.default.models import UserSocialAuth
+from mobile_api.users.serializers import CourseEnrollmentSerializer
+from openedx.core.djangoapps.user_api.api.profile import preference_info
+import serializers
+import facebook
+import json
+import urllib2
+
+# TODO: change this to final config
+from ..settings import FB_SETTINGS
+_FACEBOOK_API_VERSION = FB_SETTINGS['_FACEBOOK_API_VERSION']
+
+
+class CoursesWithFriends(generics.ListAPIView):
+    """
+    **Use Case**
+
+        API endpoint for retriving all the courses that a users friends are in. 
+        Note that only friends that allow their courses to be shared will be included.
+
+    **Example request**
+
+        GET /api/mobile/v0.5/courses/friends
+
+    **Response Values**
+
+        [{   "created": "2014-12-09 16:58:31.926438",
```

Comment:
Can we provide a link to the `UserCourseEnrollmentsList` docstring instead of copying the response structure here?  Otherwise, the doc here may not get updated when the other one does.